tox.ini

```diff
@@ -76,7 +76,8 @@ extras =
 commands =
     pip freeze --all --no-input
     oldestdeps: python -c "import astropy.time; astropy.time.update_leap_seconds()"
-    !online-!hypothesis-!figure: {env:PYTEST_COMMAND} {posargs}
+    !online-!hypothesis-!figure-!oldestdeps: {env:PYTEST_COMMAND} {posargs}
+    oldestdeps: {env:PYTEST_COMMAND} {posargs} -n 0 --dist no
```

Comment:
Do you know why xdist isn't working on oldestdeps for 4.0?

Reply:
I don't know and this is easier.

Reply:
👍 - for future reference can you leave a comment explaining why we're not using `xdist` on oldestdeps, ideally with the error message that happens when it's enabled.

Reply:
Will add.